README.md

```diff
@@ -39,56 +39,62 @@ Limber should be accessible via [http://localhost:3001](http://localhost:3001).
 
 ## Initial Setup (using native installation)
 
-Steps must be followed in either the Sequencescape repository or this Limber
-repository, as indicated:
+Because Limber relies on SequenceScape as a backend, SequenceScape and it's jobs can be started first. SequenceScape **will** work standalone without Limber.
```

Comment:
'must' be started first.
Limber **will not** work independently.
Limber is the front end for Sequencescape and handles pipeline tracking functionality used by Lab staff.